CoreUtilities/src/au/gov/asd/tac/constellation/utilities/visual/VisualManager.java

```diff
@@ -172,7 +181,52 @@ public final void startProcessing() {
      */
     public final void stopProcessing() {
         isProcessing = false;
-        processingThread.interrupt();
+        cancelProcessing(true);
+    }
+    
+    protected final void cancelProcessing(final boolean cancel) {
+        if (processingFuture != null) {
+            processingFuture.cancel(cancel);
+        }
+    }
+    
+    protected final boolean isProcessing() {
+        return isProcessing;
+    }
+    protected final boolean isRefreshProcessor() {
+        return refreshProcessor;
+    }
+    protected final boolean isIndigenousChanges() {
+        return indigenousChanges;
+    }
+    protected final void setProcessing(final boolean newVal) {
+        isProcessing = newVal;
+    }
```

Comment:
are you able to remove this setter (doesn't appear to be used anyway)

Reply:
done